src/KomaMRI.jl

```diff
@@ -1,78 +1,21 @@
 module KomaMRI
 
 #IMPORT PACKAGES
-import Base.*, Base.+, Base.-, Base./, Base.vcat, Base.size,
-       Base.copy, Base.Threads.@spawn, Base.Threads.@threads,
-       Base.angle, Base.abs, Base.getproperty, Base.one, Base.zero
-#General
-using Pkg, Random, Reexport, ThreadsX
-#Printing
-using Scanf, ProgressMeter
-#Datatypes
-using Parameters
-#Simulation
-using CUDA, Interpolations
-#Reconstruction
-using MRIReco, MRIFiles
-@reexport using MRIReco: RawAcquisitionData, AcquisitionData, reconstruction
-@reexport using MRIFiles: ISMRMRDFile
-#IO
-using FileIO, HDF5, MAT, JLD2
+using Reexport
+@reexport using KomaMRICore
+import KomaMRICore: update_blink_window_progress!
+
 #GUI
 using Blink, Interact, PlotlyJS, AssetRegistry
 @reexport using PlotlyJS: savefig
```

Comment:
Is this necessary?

Reply:
It is not necessary to reexport. However savefig() is used directly in many examples of the documentation, so it is convenient to export savefig().

Reply:
I agree, it is fine while it remains in `KomaMRIPlots`.